app/assets/javascripts/sufia/autocomplete.es6

```diff
@@ -0,0 +1,51 @@
+export class Autocomplete {
+  constructor() {
+  }
+
+  // This is the initial setup for the form.
+  setup() {
+      $('[data-autocomplete]').each((index, value) => {
+          let selector = $(value)
+          switch (selector.data('autocomplete')) {
```

Comment:
@jcoyne Seems like this code and the code at line 25 are very similar. Wondering if they could be combined so we are not testing for the same hard coded strings in two places.  This is not a show stopper for the PR by any means...